src/components/TileCatalogNew/TileCatalogNew.jsx

```diff
@@ -66,6 +66,7 @@ const defaultProps = {
     searchPlaceHolderText: 'Enter a value',
     error: 'An error has occurred. Please make sure your catalog has content.',
   },
+  tiles: null,
```

Comment:
Would it be more appropriate for this to be an empty array to match the prop types?

```suggestion
  tiles: [],
```

Reply:
To match the current implementation, it would need to be null. It would probably better to fix the conditional as it makes more sense to have an empty array rather it be null
```
{tiles && tiles.length && !error ? (
            <div className={`${iotPrefix}--tile-catalog--tile-canvas--content`}>{renderGrid()}</div>
          ) : (
            <Tile className={`${iotPrefix}--tile-catalog--empty-tile`}>
              <>
                <Bee32 />
                <p>{error || i18n.error}</p>
              </>
            </Tile>
          )}
```

Reply:
We're not bound by semver on this component, we can make that update if it makes sense. Good point on the conditional

Reply:
Went ahead and swapped this to your recommended change